dm/task-configuration-file-full.md

```diff
@@ -101,7 +101,7 @@ block-allow-list:                    # Use black-white-list if the DM version is
 # Configuration arguments of the dump processing unit.
 mydumpers:
   global:                            # The configuration name of the processing unit.
-    threads: 4                       # The number of the threads that the dump processing unit dumps from the upstream database (4 by default).
+    threads: 4                       # The number of threads that access the upstream when the dump processing unit performs the precheck and exports data from the upstream database (4 by default)
     chunk-filesize: 64               # The size of the file generated by the dump processing unit (64 in MB by default).
     extra-args: "--consistency none" # Other arguments of the dump processing unit. You do not need to manually configure table-list in `extra-args`, because table-list is automatically generated by DM.
```

Comment:
```suggestion
    extra-args: "--consistency none" # Other arguments of the dump processing unit. You do not need to manually configure table-list in `extra-args`, because it is automatically generated by DM.
```